src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -237,8 +238,16 @@ class Wedge3D {
   std::array<tt::remove_cvref_wrap_t<T>, 3> operator()(
       const std::array<T, 3>& source_coords) const noexcept;
 
+  /// For a \f$+z\f$-oriented `Wedge3D`, returns invalid if \f$z<=0\f$
+  /// or if \f$(x,y,z)\f$ is on or outside the paraboloid defined
+  /// by \f$(x^2/z^2 + y^2/z^2+1)^{1/2} = -S/F\f$, where
```

Comment:
I might have made a mistake, but I'm getting that this is a cone shape?

Reply:
Yes!  It's a cone, sorry, will fix.